packages/zipkin-instrumentation-redis/test/integrationTest.js

```diff
@@ -73,6 +73,65 @@ describe('redis interceptor', () => {
     });
   });
 
+  it('should add zipkin annotations for batch method', (done) => {
+    const ctxImpl = new ExplicitContext();
+    const recorder = {record: sinon.spy()};
+    // const recorder = new ConsoleRecorder();
+    const tracer = new Tracer({ctxImpl, recorder});
+
+    const redis = getRedis(tracer);
+    redis.on('error', done);
+    tracer.setId(tracer.createRootId());
+    const ctx = ctxImpl.getContext();
+    redis.set('ping', 'pong', 10, () => {
+      ctxImpl.letContext(ctx, () => {
+        redis.batch([ ['get', 'ping'] ]).exec(() => {
+          const annotations = recorder.record.args.map(args => args[0]);
```

Comment:
probably want to verify the span name here. Not sure, but possibly some way to encode the list of "multi-ops" into a tag aka binary annotation?